build-scripts/generate_guides.py

```diff
@@ -11,8 +11,33 @@
 BenchmarkData = collections.namedtuple(
     "BenchmarkData", ["title", "profiles", "product"])
 
-XSLT_PREFIX = os.getenv('XSLT_PREFIX', default='/usr')
-XSLT_PATH = os.path.join(XSLT_PREFIX, "share/openscap/xsl/xccdf-guide.xsl")
+OPENSCAP_POSSIBLE_ROOT_DIRS = [
+    os.getenv("OPENSCAP_ROOT_DIR"),
+    os.getenv("ProgramFiles"),
+    os.getenv("XSLT_PREFIX"),
+    "/usr",
+    "/usr/bin",
+    "/usr/sbin",
+    "/usr/local",
+    "/usr/share/",
+    "/usr/local/share",
+    "/opt",
+    "/opt/local",
+]
+
+for root_dir in OPENSCAP_POSSIBLE_ROOT_DIRS:
+    if root_dir is None:
+        continue
+    for subpath in [os.path.join("share", "openscap", "xsl"), "xsl"]:
+        file_path = os.path.join(root_dir, subpath, "xccdf-guide.xsl")
+        if os.path.exists(file_path):
+            XCCDF_GUIDE_XSL = file_path
+            break
+    else:
+        continue
+    break
+else:
+    XCCDF_GUIDE_XSL = None
```

Comment:
If we want to do things correctly, the path should be found by CMake and then passed to the script as a parameter.